receiver/hostmetricsreceiver/internal/scraper/filesystemscraper/config.go

```diff
@@ -23,15 +25,93 @@ import (
 type Config struct {
 	internal.ConfigSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct
 
-	// Include specifies a filter on the devices that should be included from the generated metrics.
-	// Exclude specifies a filter on the devices that should be excluded from the generated metrics.
-	// If neither `include` or `exclude` are set, metrics will be generated for all devices.
-	Include MatchConfig `mapstructure:"include"`
-	Exclude MatchConfig `mapstructure:"exclude"`
+	// IncludeDevices specifies a filter on the devices, filesystem types or mount points that should be included from the generated metrics.
```

Comment:
I think the comment is wrong?

```suggestion
	// IncludeDevices specifies a filter on the devices that should be included from the generated metrics.
```

Also Nit: should really be "included in" rather than "included from".